nextcord/ui/text_input.py

```diff
@@ -0,0 +1,266 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-present Rapptz
```

Comment:
would say only use epic in license like application_command.py as it is a new file
```suggestion
Copyright (c) 2021-present tag-epic
```